pkgs/by-name/hy/hyprland/package.nix

```diff
@@ -191,6 +196,9 @@ stdenv.mkDerivation (finalAttrs: {
       wozeparrot
     ];
     mainProgram = "Hyprland";
-    platforms = lib.platforms.linux;
+    platforms = lib.concatLists [
+      lib.platforms.linux
+      lib.platforms.freebsd
+    ];
```

Comment:
`lib.platforms.linux ++ lib.platforms.freebsd` would be cleaner

Reply:
Done